transport/src/test/java/io/el/channel/DefaultChannelPipelineTest.java

```diff
@@ -70,4 +81,115 @@ public void removeOnlyOne() {
       assertNull(channelPipeline.firstContext());
     }
   }
+
+  @Nested
+  @DisplayName("On bind() method")
+  class BindMethod {
+    @Test
+    public void checkChannelBindCalled() throws InterruptedException {
+      Executor executor = new ThreadPerTaskExecutor(
+          Executors.defaultThreadFactory());
+      TestEventLoop eventLoop = new TestEventLoop(executor);
```

Comment:
How about mocking EventLoop which returns true when calling its method `eventLoop.inEventLoop()`? Because the eventLoop#execute() method has lots of logic and I think it's ok to assume EventLoop already starts its thread.

Reply:
It seems hard to mock `inEventLoop`. It is hard to mock EventLoop since identifying the functions used in the bind call is difficult.

Reply:
Could you update `TestEventLoop` with `DefaultChannelEventLoop`